R/sun-methods.R

```diff
@@ -110,6 +161,10 @@ setMethod("sunriset", signature(crds="sf", dateTime="POSIXct"),
               res
           })
 
+#' @description Method for matrix objects. 
```

Comment:
These `@description` fields should be omitted from the method documentation, because they produce a line in the Description paragraph of the function documentation.